launch/make_batch_file.py

```diff
@@ -0,0 +1,9 @@
+import csv
+from typing import IO, List
+
+
+def make_batch_input_file(urls: List[str], file: IO[str]):
```

Comment:
have we committed to doing CSV for everything in the backend? i feel like json might be more extensible since we can add more info

Reply:
I don't think we've quite committed to csv, but the current implementation of pyspark works with csv (probably would be pretty easy to get it to use json though, but I'm not an expert on pyspark)

Reply:
Although my take is that the json will probably need to be very strictly structured anyways if it's gonna be the input to pyspark, e.g. a list of dicts with the same keys or something along those lines, so we won't really get much help from json's flexibility

Separate question might be "do we want the client to provide an input that conforms to this strict format", my take is that if the user passes in a list of urls (or a map {id: url}) to the client then the format can be made pretty strict